.travis-setup.sh

```diff
@@ -22,6 +22,23 @@ set -e -x
 cwd=$(cd `dirname "$0"`; pwd -P)
 source $cwd/versions.txt
 
+#
+# Install go
+#
+
+go_tarball=go${go_version}.linux-amd64.tar.gz
+curl -L -O https://storage.googleapis.com/golang/$go_tarball
```

Comment:
I'd always quote strings like this to ensure they are future-proof against weird expansions.